tests/integration/pybaseball/datasources/test_fg_pitching_data.py

```diff
@@ -120,15 +123,15 @@ def test_fg_pitching_maximum_age(self) -> None:
         assert len(data.columns) == self.ALL_DATA_COLUMNS_COUNT
         assert len(data.index) == 1
 
-    def test_fg_pitching_team(self, assert_frame_not_equal) -> None:
+    def test_fg_pitching_team(self, assert_frame_not_equal: Callable[[pd.DataFrame, pd.DataFrame], None]) -> None:
```

Comment:
Wasn't there a specific thingy above for this?

Reply:
Not for this one, but I'll add one.